src/frontend-pwa/src/views/Report/report.styles.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * @summary This is the styling page for the Report view screen.
+ * @author  Tyler Maloney
+ * */
+
+import styled from '@emotion/styled';
+import typography from '../../typography';
+
+export const StyledReportOuterDiv = styled.div`
+    width: 100%;
+    left:0;
+    top:0;
+    min-width: 100vw;
+    min-height: 80vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 30pt 0;
+    background-color: white;
+    @media (min-width: 768px) {
+        padding: 20px;
+    }
+`;
+export const StyledReportContainer = styled.div`
+    ${typography.toString()}
+    width: 100%;
+    max-width: 768px;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
```

Comment:
I would consider aligning items to the left instead of centered to give a consistent flow.
Currently you are centering all the input fields, but the text are still left aligned 
![image](https://github.com/bcgov/citz-imb-wayfinder/assets/62873746/61e012c6-4e6b-4f10-a975-7143ec7371ad)

And then make the form inputs all equal sizing across.
I did some really hacky CSS changes locally just give you a sense of what I'm getting at  (I would keep the button centered though)
![image](https://github.com/bcgov/citz-imb-wayfinder/assets/62873746/44f34859-e958-4204-9f08-d75a377abb1d)